releasenotes/notes/refactors-and-upgrades-qeom-bd14d85c94e912d0.yaml

```diff
@@ -0,0 +1,65 @@
+---
+features:
+  - |
+    Improves the QEOM code and implements the calculation of excited state properties and transition 
+    amplitudes with QEOM.
```

Comment:
Can you please make `QEOM` a link like the ones I corrected further down in the release note?